exampleRawHttpRequest/connectors/testconnector/rawHttpRequest.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+
+	method: ({ method }) => {
```

Comment:
Should there be a more concrete example?

Reply:
The example files are more to run a connector locally against the connector, and to actively test them; it's not for documentation, but development.
During development, the dev can change/update it however they want.